docs/keycloak-configuration.md

```diff
@@ -26,153 +33,313 @@ To create a new realm follow the steps below:
 
 - Click on the "Master" realm drop-down on the top left side of the console as you can see below
 - Click on "Create Realm"
-- Give the realm a realm name - For example - "tornjak"
+
+![Keycloak Administration Console](/docs/rsrc/keycloak_diagrams/6-admin-console.png)
+
+- Give the realm a realm name - e.g. "Tornjak"
 - Make sure "Enabled" is set to On, as a realm that is not enabled can not be used
-- [Option 1]: The easiest way to create a realm is by using the import feature. To create the Tornjak realm download this (link for JSON file) file. And import to Keycloak by clicking on browse and import your resource file.
+
+![Create Realm Page](/docs/rsrc/keycloak_diagrams/7-create-realm-page.png)
+
+- Click on Create
+- Once the realm is created, you will see the realm name on the top left side of the console as you can see below. Here is where you can switch between different realms you created and visualize the Master Realm settings.
+
+![Tornjak Realm Page](/docs/rsrc/keycloak_diagrams/8-realm-page.png)
+
+# Option 1: Realm Import
+
+- The easiest way to create a realm is by using the import feature. To create the Tornjak realm download [this](examples/Tornjak-keycloak-realm-import.json) file. 
+
+- And import to Keycloak by going to Realm settings. 
+
+![Tornjak Realm Settings Page](/docs/rsrc/keycloak_diagrams/9-realm-settings-page.png)
+
+- Click on Partial Import, on the top right corner under Action.
+
+![Tornjak Realm Import Page](/docs/rsrc/keycloak_diagrams/10-realm-import-page.png)
+
+- Here, you can import your realm resources by clicking on browse and importing your resource file, as you can see below.
+
+![Imported Tornjak Realm Json](/docs/rsrc/keycloak_diagrams/11-imported-realm-json.png)
+
+- Edit any resource in the window present as a JSON format and select the resources you want to import by using the checkbox below the edit window. You can choose what to do if a specific resource already exists, such as Fail Import, Skip or Overwrite. 
+
+- Click Import, and you will see the resources imported as "Added" under the Action column including the realm resource info such as the Type, Name, and ID of the resource imported.
+
+![Imported Resources](/docs/rsrc/keycloak_diagrams/12-imported-resources-window.png)
+
+- Click on Close, to close the window and check if all the resources are imported by going to the respective resource section.
+
 - If you imported all your resources using the above step skip to the appropraite section depending on which resources you already imported such as the clients, roles and groups together with the realm.
```

Comment:
the one other thing i notice is the 'Full Scope Allowed' is still given to the client, which needs to be disabled. so perhaps this should be also added here?